icaruscode/NuGraphIcarus/hdf5maker_icarus_slice.fcl

```diff
@@ -0,0 +1,92 @@
+#include "services_common_icarus.fcl"
+#include "services_icarus_simulation.fcl"
+
+process_name: trueenergy
+
+services:
+{
+  TFileService:          { fileName: "reco_hist.root" }
+  TimeTracker:           {}
+  MemoryTracker:         {}
+  RandomNumberGenerator: {}
+  FileCatalogMetadata:   @local::art_file_catalog_mc
+  @table::icarus_geometry_services
+  @table::icarus_common_services
+  @table::icarus_backtracking_services 
+  message:               @local::icarus_message_services_prod_debug
+  ParticleInventoryService: { 
+        ParticleInventory: {
+        EveIdCalculator: "EmEveIdCalculator"
+        G4ModuleLabel: "largeant"
+	OverrideRealData: true
+     }
+  }
+
+                         
+} # services
+
+source:
+{
+  module_type: RootInput
+  maxEvents:   -1
+}
+
```

Comment:
This block is actually not necessary (`RootInput` with `maxEvents` is the default when an input file is specified on the command line).
```suggestion
```